File: app/src/main/java/org/tvheadend/tvhclient/ui/features/epg/CustomHorizontalLayoutManager.java
package org.tvheadend.tvhclient.ui.features.epg;

import android.content.Context;
import androidx.recyclerview.widget.LinearLayoutManager;

class CustomHorizontalLayoutManager extends LinearLayoutManager {

    CustomHorizontalLayoutManager(Context context) {
        super(context, HORIZONTAL, false);
    }

    @Override
    public boolean canScrollHorizontally() {
        return false;
    }
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/epg/CustomHorizontalLayoutManager.kt
package org.tvheadend.tvhclient.ui.features.epg

import android.content.Context
import androidx.recyclerview.widget.LinearLayoutManager

internal class CustomHorizontalLayoutManager(context: Context) : LinearLayoutManager(context, HORIZONTAL, false) {

    override fun canScrollHorizontally(): Boolean {
        return false
    }
}
